op-service/txmgr/cli.go

```diff
@@ -472,11 +478,34 @@ func NewConfig(cfg CLIConfig, l log.Logger) (*Config, error) {
 	res.MinTipCap.Store(minTipCap)
 	res.MaxTipCap.Store(maxTipCap)
 	res.MinBlobTxFee.Store(defaultMinBlobTxFee)
-	res.EnableCellProofs = cfg.EnableCellProofs
 
 	return &res, nil
 }
 
+func fallbackToOsakaCellProofTimeIfKnown(chainID *big.Int, cellProofTime uint64) uint64 {
+	if cellProofTime != math.MaxUint64 {
```

Comment:
Wondering if we want to do

```suggestion
	if cellProofTime != defaults.CellProofTime {
```
here instead?